components/tools/OmeroPy/test/integration/clitest/test_chown.py

```diff
@@ -57,17 +58,18 @@ def testChownBasicUsageWithId(self, object_type, user_prefix):
         assert obj.id.val == oid
         assert obj.details.owner.id.val == user.id.val
 
-    def testChownBasicUsageWithName(self):
-        oid = self.create_object("Image")
+    @pytest.mark.parametrize("object_form", object_forms)
+    def testChownBasicUsageWithName(self, object_form):
+        oid = self.create_object("Dataset")
```

Comment:
As you are using a Dataset I would suggest different parametrerization here:
```
@pytest.mark.parametrize("object_prefix", ["", "/"])
@pytest.mark.parametrize("object_suffix", ["", "I"])
def testChownBasicUsageWithName(self, object_prefix, object_suffix):
...
```
then create that args using the prefic, object name and suffix